src/createServer.js

```diff
@@ -1,3 +1,41 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { validateURL } = require('./validateUrl');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const formatedPath = req.url.slice(1);
+    const [text, queryString] = formatedPath.split('?');
```

Comment:
You can use the `new URL` construction

```suggestion
    const formatedUrl = new URL(req.url, `http://${req.headers.host}`);
    const originalText = normalizedUrl.pathname.slice(1);
    const toCase = normalizedUrl.searchParams.get('toCase');
```